.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
will this file used? or this is for documentation purposes?

Reply:
This file is used by VSCode to define the default debug configurations.
![image](https://user-images.githubusercontent.com/27447/51343637-d723a500-1a64-11e9-941f-3d70140d4f88.png)